modules/SitecoreImageBuilder/SitecoreImageBuilder.psm1

```diff
@@ -148,7 +153,7 @@ function Invoke-Build
         [string]$Registry
         ,
         [Parameter(Mandatory = $false)]
-        [array]$Tags = (Get-LatestSupportedVersionTags)
+        [array]$Tags
```

Comment:
Is there a reason for moving this from a default parameters to the if statement at ~ line:187

Reply:
I moved it to reduce the number of calls to `Get-BuildSpecifications`